### ISSUE TITLE
VSCode extension updated today

### ISSUE THREAD

**Issue by bdoyle8** [microsoft/vscode#204190]
Type: <b>Bug</b>

REmote-SSH v.108.0 installed today and now I cannot connect to the server or edit my config file. I can't do anything.

VS Code version: Code 1.86.0 (05047486b6df5eb8d44b2ecd70ea3bdf775fd937, 2024-01-31T10:28:19.990Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Connection to 'SSH: Develop' could not be established 

Also, the dropdown for report an error would not allow me to pick an extension ... so I did  not.
<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 PRO 4650U with Radeon Graphics (12 x 2096)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.23GB (3.87GB free)|
|Process Argv|--crash-reporter-id 5f1c9ba4-1be8-4649-a669-f5915bbc8de6|
|Screen Reader|no|
|VM|0%|

Connection to 'SSH: Develop' could not be established 
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
vscrp:30673768
962ge761:30951796
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
dsvsc019bcf:30953938
3ef8e399:30949928
e3gdj431:30956216

```

</details>

**Comment by gjsjohnmurray** [comment]
What server OS version?

**Comment by bdoyle8** [comment]
John,

This issue can be closed. I just finished testing / verifying the fix my Linux admin suggested.  The fix outlined below we are rolling out to the organization, since the servers cannot be updated for the new libraries at this time.  We also had to disable "enable windows background updates" to prevent VSCode from updating to the most current version.

I do wish the new version was compatible with the libraries like v1.85.2 is compatible.

VS Code version was 1.86.  I have just finished worked with our linux server team and here is what we found out: Linux servers have RHEL version 7.9 and it does not have the version of libstdc++ that this version of VSCode requires.  Per my linux admin, I installed a prior version and no longer have the issue:
Version: 1.85.2 (user setup)
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:10.514Z
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19045

I went to this site to get the prior version: https://code.visualstudio.com/updates/v1_85

Barbara Doyle
Senior Professional Application Designer
MS OPS / Buy-In TFAL / DSS
T +1.608.684.0236
***@***.******@***.***>

Planned PTO:  None scheduled at this time
 ***@***.***
